aksharamukha/transliterate.py

```diff
@@ -517,6 +518,43 @@ def convert_default(src, tgt, txt, nativize = True, post_options = [], pre_optio
 
     return convert(src, tgt, txt, nativize, pre_options, post_options)
 
+def convert_docx(src, tgt, file, nativize, pre_options, post_options):
+    new_file = io.BytesIO()      
+
+    with ZipFile(file, 'r') as old_archive:
+        with ZipFile(new_file, 'w') as new_archive:
+            for item in old_archive.filelist:
+                if item.filename.startswith('word/') and item.filename.endswith('.xml'):
+                    with old_archive.open(item, 'r') as xml_file:
+                        tree = etree.parse(xml_file)
+                        for element in tree.getroot().iter():
+                            if element.tag == "{http://schemas.openxmlformats.org/wordprocessingml/2006/main}t":
+                                element.text = convert(
+                                    src, tgt, element.text, nativize, post_options, pre_options)
+
+                        string = etree.tostring(
+                        tree.getroot(), xml_declaration=True, encoding="UTF-8", standalone=True)
+
+                        new_archive.writestr(item, string)
+
+                else:
+                    # Copy other contents as it is
+                    new_archive.writestr(item, old_archive.read(item.filename))
+
+    if(isinstance(file, str)):
+        # python package use
+        # Flush new zip to the file
```

Comment:
I'd consider the "comment to telling method name" refactoring, i.e. instead of writing a comment which explains what the code is doing, just create a function whose name describes what's going on. This applies here as well as for the other comments.

```suggestion
        self.flush_output_zip_to_file(file)
```